XPlat/TensorFlow.Lite/cake.packages.config

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Again, let's let the root folder scripts take care of this, and delete this one.